thor/params.go

```diff
@@ -38,6 +38,10 @@ const (
 
 	SeederInterval     = 8640 // blocks between two seeder epochs.
 	CheckpointInterval = 180  // blocks between two bft checkpoints.
+
+	ElasticityMultiplier     = 2
+	InitialBaseFee           = 1000000000
+	BaseFeeChangeDenominator = 8
```

Comment:
nit: Can some comments be added here ?

Reply:
I have them in the third and final PR, so they are definitive